tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -258,6 +240,14 @@ var _ = Describe("odo devfile create command tests", func() {
 		})
 	})
 
+	Context("When executing odo create with component with no devBuild command", func() {
+		It("should successfully create the devfile component", func() {
+			// Quarkus devfile has no devBuild command
+			output := helper.CmdShouldPass("odo", "create", "java-quarkus")
+			helper.MatchAllInOutput(output, []string{"Please use `odo push` command to create the component with source deployed"})
```

Comment:
we should also check that run action string is present and build action string is not present.

Reply:
> we should also check that run action string is present and build action string is not present.

But this test is only for `odo create` verification and I don't find anything related to run and build in `odo create java-quarkus`. I think this should be verified in `test-cmd-devfile-push` as it is purely related to that. Logs for `odo create java-quarkus`

```
$ odo preference view
PARAMETER             CURRENT_VALUE
UpdateNotification    false
NamePrefix            
Timeout               
PushTimeout           
Experimental          true
PushTarget            

$ odo create java-quarkus
Experimental mode is enabled, use at your own risk

Validation
 ✓  Checking devfile existence [87285ns]
 ✓  Checking devfile compatibility [160078ns]
 ✓  Creating a devfile component from registry: RegistryName [165146ns]
 ✓  Validating devfile component [575155ns]

Please use `odo push` command to create the component with source deployed
```

Reply:
OK. I thought it was executing push too.

>  I think this should be verified in test-cmd-devfile-push as it is purely related to that. Logs for odo create java-quarkus

Do we really need an extra create test for `no build components`? I mean the test in `test-cmd-devfile-push` would execute the same thing too.